node/src/service_aura.rs

```diff
@@ -0,0 +1,748 @@
+// Copyright 2020-2022 Manta Network.
+// This file is part of Manta.
```

Comment:
Can you create a folder like rpc/chain_specs, then move service code to it?
Of cause, we can do it in next release.

Reply:
This `service_aura` file is temporary and we'll be getting rid of it